src/main/java/net/fabricmc/yarn/constants/SetBlockStateFlags.java

```diff
@@ -0,0 +1,46 @@
+package net.fabricmc.yarn.constants;
```

Comment:
To reduce confusion I'd suggest moving this to another source set. Putting this in main is a bit confusing.

Reply:
Done.